chain/src/sumtree.rs

```diff
@@ -515,17 +501,18 @@ impl<'a> Extension<'a> {
 		}
 	}
 
+
 	/// Current root hashes and sums (if applicable) for the UTXO, range proof
 	/// and kernel sum trees.
 	pub fn roots(
 		&self,
 	) -> (
-		HashSum<SumCommit>,
-		HashSum<NoSum<RangeProof>>,
-		HashSum<NoSum<TxKernel>>,
+		Hash,
+		Hash,
+		Hash,
```

Comment:
Should we introduce a `Root` struct to group those 3 hashes? Before the type system would prevent us from getting mixed up as to which is which but now it will be more confusing. A struct with 3 properties could help?